docs/start/framework/data-loading.md

```diff
@@ -124,10 +126,58 @@ export async function loader({ params }: Route.LoaderArgs) {
 }
 
 export async function clientLoader({
+  serverLoader,
+  params,
+}: Route.ClientLoader) {
+  const serverData = await serverLoader();
+  const res = await fetch(`/api/products/${params.pid}`);
```

Comment:
This feels redundant?  In remix we showed off the common use cases in a [dedicated page](https://remix.run/docs/en/main/guides/client-data) so maybe we pick what we think is the most common (presumably "skip the hop") and inline it into this doc but then we copy over the Remix doc and link off to it for advanced use cases?

Reply:
Yeah agreed. Initially I was combining that doc with the full stack data doc we had, but then realize we already had this doc

I think a dedicated advanced data loading explanation or how-to is a good idea. I’ll try to put something together, even if it’s just a starting point we can improve upon later